crates/core/src/db/models.rs

```diff
@@ -1293,3 +1305,43 @@ pub struct InsBufferBundleInsKey<'a> {
     /// True if the `pubkey` can be loaded as a read-write account.
     pub is_writable: bool,
 }
+
+/// Union of `listing_receipts` and `purchase_receipts` for an auction house `Activity`
+#[derive(Debug, Clone, Queryable, QueryableByName)]
```

Comment:
We don't need another model for Activity this can be the existing NftActivity. What we do need to do is add field to NFT activity for the nft. This should be dataloader based on metadata address and respond with nft objects so caller gets access to all the info they may want from the nft not lock into just the image, description.

```
activities(auctionHouses: []. limit: 10, offset: 0) {
  nft {
    image
    creators {
      ...
    }
    ...
  }
  price
  activity_type
}
```

This will be 2 queries but right now select then join is same number of select statements but 1 query submitted to the api. Like to try this because it uses existing objects which leads to consistent api.